docs/getting_started.md

```diff
@@ -54,20 +61,21 @@ Once you have the dependencies satisfied run `pip install healthcareai` or `sudo
 
 ### Verify Installation
 
-To verify that *healthcareai* installed correctly, open a terminal and run `python`. This opens an interactive python console (also known as a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)). Then enter this command: `from healthcareai import SupervisedModelTrainer` and hit enter. If no error is thrown, you are ready to rock.
+To verify that *healthcareai* installed correctly:
+1. Open a terminal and run `python` or `ipython`. Either of these opens an interactive python console (also known as a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)).
+2. Then enter this command: `from healthcareai import SupervisedModelTrainer` and hit enter. If no error is thrown, you are ready to rock.
```

Comment:
If I try "from healthcareai import SupervisedModelTrainer" I get an import error: "cannot import name 'SupervisedModelTrainer'".  On the other hand, there  is a file in healthcareai called supervised_model_trainer.py and running "from healthcareai import supervised_model_trainer" works fine.

I thought that something must have been renamed, but looking at the supervised_model_trainer file, I see that the class is indeed SupervisedModelTrainer, so I'm not sure what is going on.